### PR TITLE
chore(pipeline): Correct error in pipeline for tags and master branch

### PR DESCRIPTION
**Pull request by rockambole86**
### Background

Fix error when executing pipelines for master branch when tag already created

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -55,7 +55,7 @@ pipeline {
         stage('Publish final version images') {
             when {
                 expression {
-                    return isVersionTag(readCurrentTag())
+                    return isVersionTag(${TAG_NAME})
```
Comment (andresmoschini) [inline Jenkinsfile:58]: It does not work, what do you think about this:

```
def String readCurrentTag() {
    return sh(returnStdout: true, script: "echo ${TAG_NAME}").trim()
}
```

Comment (rockambole86) [reply Jenkinsfile:58]: yep, lemme try it

File: Jenkinsfile [hunk 2]
```diff
@@ -94,5 +94,5 @@ def CHANGE_ID = env.CHANGE_ID
 // workaround https://issues.jenkins-ci.org/browse/JENKINS-55987
 // TODO: read this value from Jenkins provided metadata
 def String readCurrentTag() {
-    return sh(returnStdout: true, script: "git describe --tags --match v?*.?*.?* --abbrev=0 --exact-match || echo ''").trim()
+    return sh(returnStdout: true, script: "echo ${TAG_NAME}").trim()
```
Comment (andresmoschini) [inline Jenkinsfile:97]: Nope.

Could you try it?

```suggestion
    return sh(returnStdout: true, script: 'echo ${TAG_NAME}').trim()
```

Comment (rockambole86) [reply Jenkinsfile]: sure... trying right now

Comment (andresmoschini) [reply Jenkinsfile]: seems to be working!

Comment (rockambole86) [reply Jenkinsfile]: yep... merging this one to create a new tag and check if problem is solved

File: Jenkinsfile (hunk 1 above)
Comment (rockambole86) [reply Jenkinsfile]: marking this as solved

File: Jenkinsfile (hunk 2 above)
Comment (rockambole86) [reply Jenkinsfile]: marking this as solved

**Pull request merged**